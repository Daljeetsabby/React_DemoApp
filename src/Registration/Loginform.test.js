import React from 'react';
import {shallow} from 'enzyme';
import Formiklogin from './loginform'

describe('Login Form', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Formiklogin/>);
    });

    test('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
