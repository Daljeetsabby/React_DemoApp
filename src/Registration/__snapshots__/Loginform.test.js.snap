// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Form renders correctly 1`] = `
<Formik
  handleSubmit={[Function]}
  initialValues={
    Object {
      "emailId": "",
      "password": "",
    }
  }
  onSubmit={[Function]}
  validationSchema={[Function]}
>
  <Component />
</Formik>
`;
